kolena/_extras/metrics/sklearn.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2021-2023 Kolena Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+try:
+    import sklearn.metrics as sklearn_metrics
+
+    sklearn_metrics.__name__
+except ImportError:
+    raise ImportError("Package 'scikit-learn' not found; install 'metrics' extras with `pip install 'kolena[metrics]'`")
```

Comment:
Are the quotes around `kolena[metrics]` required?

Reply:
maybe not - I think at some point I ran into an error (didn't appear again) asking me to use quotes or `kolena\[metrics\]`